src/components/UI/Input/index.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import styles from './styles.module.scss';
+import { InputHTMLAttributes } from "react";
+
+interface InputProps extends InputHTMLAttributes<HTMLInputElement>{}
+
+export function Input({...rest}: InputProps){
+    return(
+        <input className={styles.input} {...rest}/>
+    )
+}
```

Comment:
We could already prepare the form configs, define which form we are gonna use, and create input to use into this form. 

Suggestion: hook form.